doc/api/os.md

```diff
@@ -330,6 +330,18 @@ changes:
 Returns the operating system's default directory for temporary files as a
 string.
 
+On macOS, the returned path is usually a symlink. To get the resolved path, use
```

Comment:
usually? Can we document the conditions where it either is or isn't?

Reply:
> Can we document the conditions where it either is or isn't?

I'll replace "usually" with "by default".

Reply:
```suggestion
By default on macOS, the returned path is a symlink. To get the resolved path, use
```